src/routes/resources/[resourceContentId]/VersionStatusHistorySidebar.svelte

```diff
@@ -33,18 +34,9 @@
                 {:else}
                     <div class="flex justify-center text-lg font-semibold" dir="auto">History</div>
                     {#each fetched as statusHistory (statusHistory)}
-                        {@const date = new Date(statusHistory.dateOfEvent + 'Z')}
                         <div>
                             <div class="text-gray-500">
-                                {date.toLocaleDateString('en-us', {
-                                    month: 'short',
-                                    day: '2-digit',
-                                    year: 'numeric',
-                                    hour: '2-digit',
-                                    minute: '2-digit',
-                                    second: '2-digit',
-                                    hour12: true,
-                                })}
+                                {formatUtcToLocalTimeAndDate(statusHistory.dateOfEvent)}
```

Comment:
format from this utility function is `{H:MM} [AM | PM] {Date}`.

Reply:
We shouldn't be mixing and matching formatting in the UX anyway. The ticket should have used a consistent format. So yes, use the existing.

Reply:
The ticket format is a common display format. If the codebase doesn't have a util fn for it, I would add it.

Reply:
The plus I see to the ticket formatting is that there are seconds, and with a lot of things happening at once at times I suppose having the time be more granular would help, so that there aren't a bunch of time values that look the same, when they're really a few seconds apart

Reply:
The seconds don't matter. If they want to do an audit of the admin and look at every place we're displaying a timestamp and understand why it's a unique case that needs a specific one, they can do that. But in the meantime, we're not going to have a bunch of different utility functions formatting timestamps slightly differently.